src/delimiter.js

```diff
@@ -152,11 +152,24 @@ const makeLargeDelim = function(delim,
     return span;
 };
 
+const innerWidth = {
+    // advance width of glyphs in the KaTeX fonts.
+    "\u2016": 0.778,
+    "\u2223": 0.333,
+    "\u2225": 0.556,
+    "\u239c": 0.875,
+    "\u239f": 0.875,
+    "\u23a2": 0.667,
+    "\u23a5": 0.667,
+    "\u23aa": 0.889,
+    "\u23d0": 0.667,
```

Comment:
We should probably import fontMetricsData and look up the values from there directly, that way these numbers can't get out of sync if the fonts get updated.

Reply:
I'll do that.

But keep in mind that if the fonts change, then the paths in `svgGeomtry.js` will also be invalid. This PR, and some other SVG tactics, bind us pretty tightly to the existing font glyphs.